### DISCUSSION TITLE
How to build MT Library in Windows?

### DISCUSSION THREAD

**Discussion by hikerLi** [open-telemetry/opentelemetry-cpp#1435] [Q&A]
build-vs2019.cmd build windows libs is MD, But I want libraries of MT type.

**Comment by owent** [comment]
Maybe you can set `-DCMAKE_MSVC_RUNTIME_LIBRARY="MultiThreaded$<$<CONFIG:Debug>:Debug>"` or use `windows-static` of vcpkg.

See more https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html and https://github.com/microsoft/vcpkg/blob/master/docs/users/triplets.md for more details.
